src/math/bakhshali_sqrt.rs

```diff
@@ -0,0 +1,44 @@
+#[allow(path_statements, unused_must_use)]
```

Comment:
are these necessary?

Reply:
Kinda Of, Deleting Them Cause Some Warnings

Reply:
After Check I Found That They Actually Do Nothing :-)